README.md

```diff
@@ -228,8 +221,10 @@ See [libpg_query](https://github.com/pganalyze/libpg_query/blob/13-latest/README
 
 ## License
 
-Copyright (c) 2015, pganalyze Team <team@pganalyze.com><br>
-pg_query is licensed under the 3-clause BSD license, see LICENSE file for details.
+PostgreSQL server source code, used under the [PostgreSQL license](https://www.postgresql.org/about/licence/).<br>
+Portions Copyright (c) 1996-2023, The PostgreSQL Global Development Group<br>
+Portions Copyright (c) 1994, The Regents of the University of California
 
-Query normalization code:<br>
-Copyright (c) 2008-2015, PostgreSQL Global Development Group
+All other parts are licensed under the 3-clause BSD license, see LICENSE file for details.<br>
+Copyright (c) 2015, Lukas Fittl <lukas@fittl.com><br>
+Copyright (c) 2016-2023, Duboce Labs, Inc. (pganalyze) <team@pganalyze.com>
```

Comment:
Note this was changes to match how this looks in libpg_query, reflecting recent copyright years for both Postgres and the pganalyze-authored code portions.